docs/src/main/tut/parsing.md

```diff
@@ -60,4 +60,16 @@ import cats.syntax.either._
 val json: Json = parse(rawJson).getOrElse(Json.Null)
 ```
 
+## Warnings and known issues
+
+For Scala.js, the result of parsing JSON is restricted to the expressiveness of
+the JavaScript types. In particular numerical values may [loose precision][#393]
+when decoded by [`JSON.parse`][json.parse] to a JavaScript number. For example:
```

Comment:
OK, I've tried to clarify further. On Scala.js `Long`s are represented with [high/low pair](https://www.scala-js.org/api/scalajs-library/latest/#scala.scalajs.runtime.RuntimeLong) to ensure the Scala semantics and `StringLike.toLong` and `java.lang.long.parseLong` works correctly for the issue mentioned in #393. My understanding is that JSON.parse coerces the value into a double at which point then circe's JSON AST converter cannot do anything about it.